src/relacion-devoluciones/relacion-devoluciones.service.ts

```diff
@@ -0,0 +1,110 @@
+import { HttpException, Injectable } from '@nestjs/common';
+import { Model } from 'mongoose';
+import { InjectModel } from '@nestjs/mongoose';
+import { RelacionDevoluciones } from './schemas/relacion-devoluciones.schema';
+import { RelacionDevolucionesDto } from './dto/relacion-devoluciones.dto';
+import { FilterDto } from '../shared/dto/filter.dto';
+import { FiltersService } from '../shared/filters/filters.service';
+import { EstadosService } from '../estados/estados.service';
+import { EstadosDto } from '../estados/dto/estados.dto'
+
+@Injectable()
+export class RelacionDevolucionesService {
+  constructor(@InjectModel(RelacionDevoluciones.name) private readonly RelacionDevolucionesModel: Model<RelacionDevoluciones>,
+  private estadosService: EstadosService,
+  ) { }
+
+  async post(relacionDevolucionesDto: RelacionDevolucionesDto) {
+    try {
+      const relacionDevoluciones = new this.RelacionDevolucionesModel(relacionDevolucionesDto);
+      relacionDevoluciones.Fecha_creacion = new Date();
+      relacionDevoluciones.Fecha_modificacion = new Date();
```

Comment:
Dado que se crea y modifica al mismo tiempo, para evitar la sutil diferencia...
```suggestion
      const now = new Date();
      relacionDevoluciones.Fecha_creacion = now;
      relacionDevoluciones.Fecha_modificacion = now;
```